packages/ragbits-document-search/src/ragbits/document_search/_main.py

```diff
@@ -141,8 +141,27 @@ async def ingest(
         elements = await self.processing_strategy.process_documents(
             documents, self.document_processor_router, document_processor
         )
+        await self.__remove_entries_with_same_sources(elements)
         await self.insert_elements(elements)
 
+    async def __remove_entries_with_same_sources(self, elements: list[Element]) -> None:
```

Comment:
Is there a reason we use name mangling here, instead of marking it as private, as usual?

Reply:
No specific reason. I thought it'll be better, but can switch it to single `_`

Reply:
Ok, done